contracts/src/chainlink/cairo/ocr2/validator.cairo

```diff
@@ -46,20 +37,13 @@ func constructor{
 end
 
 @external
-func validate{
-    syscall_ptr : felt*,
-    pedersen_ptr : HashBuiltin*,
-    range_check_ptr,
-}(
-    prev_round_id: felt,
-    prev_answer: felt,
-    round_id: felt,
-    answer: felt
-) -> (valid: felt):
+func validate{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+    prev_round_id : felt, prev_answer : felt, round_id : felt, answer : felt
+) -> (valid : felt):
```

Comment:
I think `cairo-fmt` is still pretty dumb, I find this a lot less readable than the original formatting

Reply:
While perhaps I'd agree, I also appreciate the KISS approach of everybody using the same formatter and just having consistency all-round. I suggest we use it, and hope it improves with time. The next upgrade brings syntax changes to the Cairo lang so we'll see how that goes.